ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -428,12 +428,13 @@ final class SandboxServer(
         userManagementStore = userManagementStore,
         commandDeduplicationFeatures = CommandDeduplicationFeatures.of(
           Some(
-            DeduplicationPeriodSupport.of(
-              offsetSupport = DeduplicationPeriodSupport.OffsetSupport.OFFSET_NOT_SUPPORTED,
-              durationSupport = DeduplicationPeriodSupport.DurationSupport.DURATION_NATIVE_SUPPORT,
+            CommandDeduplicationPeriodSupport.of(
+              offsetSupport = CommandDeduplicationPeriodSupport.OffsetSupport.OFFSET_NOT_SUPPORTED,
+              durationSupport =
+                CommandDeduplicationPeriodSupport.DurationSupport.DURATION_NATIVE_SUPPORT,
             )
           ),
-          ParticipantDeduplicationSupport.PARTICIPANT_DEDUPLICATION_SUPPORTED,
+          CommandDeduplicationType.ASYNC_ONLY,
```

Comment:
```suggestion
          CommandDeduplicationType.SYNC_ONLY,
```

Reply:
Yup, at least we know the feature descriptors work as expected based on all the failed tests.